pkg/networkmanager/wireguard.jsx

```diff
@@ -324,6 +348,22 @@ export function WireGuardDialog({ settings, connection, dev }) {
                                     id={idPrefix + '-allowedips-peer-' + i}
                                 />
                             </FormGroup>
+                            <FormGroup className='pf-m-6-col-on-md' label={_("Preshared key")} fieldId={idPrefix + '-presharedkey-peer-' + i} labelIcon={
```

Comment:
I think the usual spelling is "pre-shared key"

Reply:
Yes :) done.